tests/test_document.py

```diff
@@ -121,7 +122,9 @@ def test_document_is_valid_when_using_or_later_licenses(self):
         file1.add_lics(lic1)
 
         package.add_lics_from_file(lic1)
-        package.add_file(file1)
+        doc.add_file(file1)
+        relationship = package.spdx_id + " " + "CONTAINS" + " " + file1.spdx_id
```

Comment:
Can we extract a helper function for this (if it does not exist already)?

Reply:
done